port/win32/omrintrospect.c

```diff
@@ -307,46 +307,9 @@ setup_native_thread(J9ThreadWalkState *state, CONTEXT *sigContext)
 		 * current thread. It seems to work, however, if we populate the EIP and EBP context
 		 * data manually (which is what the assembler does below).
 		 */
-#ifdef WIN64
```

Comment:
Perhaps the comment above should be updated given the removal of the assembly code?

Reply:
Thanks :) updated.